games-action/betterspades/betterspades-0.1.6~git.recipe

```diff
@@ -0,0 +1,94 @@
+SUMMARY="Ace of Spades client for low-end systems"
+DESCRIPTION="BetterSpades is an Ace of Spades client targeted at low end \
+systems (GL/ES 1.1). Runs on your grandmother's rig!
+
+* Replicates of the great game Ace of Spades (classic voxlap)
+* Runs on very old systems back to OpenGL 1.1 (OpenGL ES support too)
+* Shares similar if not even better performance to voxlap
+* Can run on "embedded" systems like a Steam Link
+* Open for future expansion
+* Easy to use
+"
+HOMEPAGE="https://github.com/xtreme8000/BetterSpades"
+COPYRIGHT="2016-2022 xtreme8000 and contributors"
+LICENSE="GNU GPL v3"
+REVISION="1"
+srcGitRev="5438d7bbd0f6a6cd7565c5cc2118fc4935205a3c"
+SOURCE_URI="https://github.com/win8linux/BetterSpades/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="bf1ea77c57eb983149e51c961b126f50c5a89fadb1a58b76f031625c62bd1135"
+SOURCE_DIR="BetterSpades-$srcGitRev"
+ADDITIONAL_FILES="betterspades.rdef.in"
+GLOBAL_WRITABLE_FILES="
+	settings/BetterSpades/config.ini auto-merge
+	"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	betterspades$secondaryArchSuffix = $portVersion
+	app:BetterSpades = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libenet$secondaryArchSuffix
+	lib:libGL$secondaryArchSuffix
+	lib:libglew$secondaryArchSuffix
+	lib:libglfw$secondaryArchSuffix
+	lib:libGLU$secondaryArchSuffix
+	lib:libopenal$secondaryArchSuffix
+	lib:libdeflate$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libenet$secondaryArchSuffix
+	devel:libGL$secondaryArchSuffix
+	devel:libglew$secondaryArchSuffix
+	devel:libglfw$secondaryArchSuffix
+	devel:libGLU$secondaryArchSuffix
+	devel:libopenal$secondaryArchSuffix
+	devel:libdeflate$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:cmake
+	cmd:git
+	cmd:gcc$secondaryArchSuffix
+	cmd:make
+	cmd:pkg_config
+	cmd:unzip
+	"
+
+BUILD()
+{
+	cd build/
+	cmake .. \
+		-DCMAKE_BUILD_TYPE=RelWithDebInfo \
+		-DCMAKE_EXE_LINKER_FLAGS=-lnetwork
+	make $jobArgs
+}
+
+INSTALL()
+{
+	mkdir -p $appsDir/BetterSpades
+	mkdir -p $settingsDir/BetterSpades
+	cp -r build/BetterSpades/* $appsDir/BetterSpades
+	mv $appsDir/BetterSpades/config.ini $settingsDir/BetterSpades/config.ini
+	mv $appsDir/BetterSpades/cache $settingsDir/BetterSpades/cache
+	rm -f $settingsDir/BetterSpades/cache/.gitignore
+	
```

Comment:
Trailing whitespace here (strange that it doesn't show up on github)

Reply:
Weird that Pe didn't trim that, even though it is set to trim trailing whitespaces. Fixing it locally right now.